test/TestUtilities/Test.Utility/JsonData.cs

```diff
@@ -4765,5 +4765,21 @@ public static class JsonData
 }}";
 
         #endregion
+
+        #region AutoCompleteResourceV3Index
+        public static string AutocompleteResourceV3Index = @"{
```

Comment:
This is not quite right (although it does test your change).

You should test against the actual content of the link you'd hit. 

Namely this should be the content of:
https://api-v2v3search-0.nuget.org/autocomplete?q=newt&prerelease=true&semVerLevel=2.0.0

Furthermore, I don't think the name is right. 
It's not the AutocompleteResourceV3Index. It should the autocomplete endpoint newt result.

Reply:
Thanks! Will fix it.